### PR TITLE
#9 - showing current tile

### PR DESCRIPTION
**Pull request by krzysztofrewak**
This is some old pull request I worked on. @BarTracz, please check how it works and review it.

### CODE REVIEW

**Comment by BarTracz** [comment]
image
For me it looks good so i guess we can merge it

File: index.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
```
Comment (krzysztofrewak) [inline index.php:1]: Do you think it's a good idea to put this file here? Won't it change with every commit? 

Maybe creating an empty example / skeleton would be better?

**Review by AleksandraKozubal — COMMENTED** [review]
I think it would be nice to add ports and command section, allowing access to app via localhost. For example 
`ports:
      - "8080:8080"`
 `command: php -S 0.0.0.0:8080 -t /application`
    
Maybe we could add a simple description of the dev setup steps?

**Comment by krzysztofrewak** [comment]
> I think it would be nice to add ports and command section, allowing access to app via localhost. For example 
> 
> `ports:
> 
>       - "8080:8080"`
> 
>  `command: php -S 0.0.0.0:8080 -t /application`
> 
>     
> 
> Maybe we could add a simple description of the dev setup steps?

Do we really need this in library repository?

**Comment by AleksandraKozubal** [comment]
> > I think it would be nice to add ports and command section, allowing access to app via localhost. For example
> > `ports:
> > ```
> >   - "8080:8080"`
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > `command: php -S 0.0.0.0:8080 -t /application`
> > Maybe we could add a simple description of the dev setup steps?
> 
> Do we really need this in library repository?

I thought of it as a dev way of presenting (especially useful for code reviews) that would be removed after the work was finished - but sure, I won't insist on it

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,6 @@
 .idea
 .composer
+.phpunit.result.cache
 vendor
 composer.lock
+index.php
```
Comment (krzysztofrewak) [inline .gitignore:6]: You need to remove this entry from this file, push your changes, then remove index file, push changes again and then add this line to git ignore.

Comment (BarTracz) [reply .gitignore:6]: Done

**Review by BarTracz — DISMISSED** [review]

**Review by kamilpiech97 — APPROVED** [review]

**Pull request merged**